modules/core/src/main/java/org/apache/ignite/internal/processors/configuration/distributed/DistributedThinClientConfiguration.java

```diff
@@ -25,21 +25,36 @@
 
 import static org.apache.ignite.internal.cluster.DistributedConfigurationUtils.makeUpdateListener;
 import static org.apache.ignite.internal.processors.configuration.distributed.DistributedBooleanProperty.detachedBooleanProperty;
+import static org.apache.ignite.internal.processors.configuration.distributed.DistributedIntegerProperty.detachedIntegerProperty;
 
 /**
  * Thin client distributed configuration.
  */
 public class DistributedThinClientConfiguration {
-    /** */
-    private final IgniteLogger log;
+    /** Default connection count limit. */
+    public static final int DFLT_MAX_CONNECTIONS_PER_NODE = 0;
+
+    /** Message of baseline auto-adjust parameter was changed. */
```

Comment:
Baseline auto-adjust?

Reply:
Yeah, that's weird, but it was like this before me (you can see in red below) :). Fixed.